x-pack/test/security_solution_cypress/config.ts

```diff
@@ -46,6 +46,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         '--xpack.ruleRegistry.unsafe.legacyMultiTenancy.enabled=true',
         `--xpack.securitySolution.enableExperimental=${JSON.stringify([
           'alertSuppressionForNewTermsRuleEnabled',
+          'alertSuppressionForNonSequenceEqlRuleEnabled',
```

Comment:
As the FF has been activated here, all the tests are going to be affected by it, not only the ones where the FF has specifically added at the top of the spec file.

Reply:
As far as I'm aware, this is still the only way to run a cypress test locally with a feature flag enabled (https://github.com/elastic/security-team/issues/7423). Please correct me if that's changed.

The global scope is unfortunate, but we don't really have an alternative. If the flag is not enabled globally, then a developer attempting to run a FF test would see it pass on CI and then fail locally; how would they know to set the global config in order to see the test pass?

Reply:
You are completely right. 

If I'm not mistaken that behaviour should be documented so everybody should be aware about it. I understand your point, if you prefer to have it globally then you can remove the FF from the spec files.

My comment was more a reminder about what the behaviour is going to be :)

Reply:
>  if you prefer to have it globally

I would certainly prefer _not_ to set the global configuration, as it's both onerous and broad. But until https://github.com/elastic/security-team/issues/7423 is addressed, it's the only way to develop cypress tests requiring a feature flag.